### ISSUE TITLE
BuildAnalysis does not work with new .NET 6 WPF projects

### ISSUE THREAD

**Issue by hailedev** [mrlacey/Rapid-XAML-Toolkit#518]
### Describe the bug

When the RapidXaml.BuildAnalysis package is installed into a project created from the .NET 6 WPF Application template the analysis executable is not run when the application is built

### How can this be reproduced?

1. Open Visual Studio 2022, create a new project and select the WPF Application template, select all default options in the wizard
2. Add the RapidXaml.BuildAnalysis package to the solution
3. In the xaml file add a tag which would generate a warning such as a TextBlock with a hardcoded string
4. Build the solution

Expected result: a warning is generated in the build output

Have created a video to demo the problem below

https://user-images.githubusercontent.com/6165355/213951900-68cb5511-863f-4f36-a48e-1fef3805eee2.mp4

### Which version of Visual Studio are you using?

- [ ] 2019
- [X] 2022

### Project Type

WPF

**Comment by hailedev** [comment]
This looks to be because `EnableDefaultItems` is enabled in the project template. You need to update the csproj file to disable it and then explictly add in the xaml files

**Comment by mrlacey** [comment]
Yes, this is due to the way that files to analyze are identified.
Now I know you're hitting this issue, I'll bump it up the priority list. 

Note, this also affects MAUI apps too :(

**Comment by mrlacey** [comment]
x-Ref: #408
